crates/rbuilder/src/utils/error_storage.rs

```diff
@@ -83,10 +83,10 @@ pub fn store_error_event<T: serde::Serialize>(category: &str, error: &str, paylo
                 return;
             }
         };
-        if payload_json.as_bytes().len() > MAX_PAYLOAD_SIZE_BYTES {
```

Comment:
Same as before, is this change required?

Reply:
`make lint` error again